src/main/java/uk/gov/justice/hmpps/prison/api/resource/UserResource.java

```diff
@@ -91,7 +91,7 @@ public List<CaseLoad> getMyCaseLoads(@RequestParam(value = "allCaseloads", requi
             @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "List of all case note types (with sub-types) accessible to current user (and based on working caseload).", description = "List of all case note types (with sub-types) accessible to current user (and based on working caseload).")
     @GetMapping("/me/caseNoteTypes")
-    public List<ReferenceCode> getMyCaseNoteTypes(@RequestHeader(value = "Sort-Fields", required = false) @Parameter(description = "Comma separated list of one or more of the following fields - <b>code, activeFlag, description</b>") final String sortFields, @RequestHeader(value = "Sort-Order", defaultValue = "ASC", required = false) @Parameter(description = "Sort order (ASC or DESC) - defaults to ASC.") final Order sortOrder) {
+    public List<ReferenceCode> getMyCaseNoteTypes() {
```

Comment:
was all those parameters not being used?

Reply:
yep